src/test/java/org/broadinstitute/consent/http/service/DatasetServiceTest.java

```diff
@@ -126,6 +127,20 @@ public void testFindInvalidProperties() {
         Assert.assertFalse(properties.isEmpty());
     }
 
+    @Test
+    public void testFindDuplicateProperties() {
+        initService();
+
+        List<DataSetPropertyDTO> input = getDatasetPropertiesDTO();
+        DataSetPropertyDTO duplicateProperty = input.get(0);
+        input.add(duplicateProperty);
+
+        List<DataSetPropertyDTO> properties = datasetService.findDuplicateProperties(input);
+
+        Assert.assertFalse(properties.isEmpty());
+        Assert.assertEquals(properties.get(0), duplicateProperty);
```

Comment:
minor nit: I like to use the static imports so it's just:
```suggestion
        assertEquals(properties.get(0), duplicateProperty);
```